blips/blip-2-researchcertificate.md

```diff
@@ -1,6 +1,6 @@
 # Research Object Certification
 
-| BLIP:     | 0001                                                         |
+| BLIP:     | 2                                                         |
```

Comment:
I thought it would make sense to have BLIP-1 reserved for a BLIP about BLIPs :smile: